src/protocols/bdx/AsyncTransferFacilitator.cpp

```diff
@@ -183,19 +183,23 @@ void AsyncResponder::NotifyEventHandled(const TransferSession::OutputEventType e
 
     // If this is the end of the transfer (whether a clean end, or some sort of error condition), ensure
     // that we destroy ourselves after unwinding the processing loop in the ProcessOutputEvents API.
-    // We can ignore the status for these messages since the state machine is in a bad, unrecoverable
-    // state and we should stop processing events and clean up.
+    // We can ignore the status for output events because none of them are supposed to result in
```

Comment:
```suggestion
    // We can ignore the status for these output events because none of them are supposed to result in
```
Yes, I know, you just took my suggestion....  I make mistakes too.  ;)